.drone.yml

```
@@ -17,11 +17,12 @@ steps:
   - name: test
     image: node:18
     commands:
-      - npm ci
-      - npx playwright install chromium --with-deps
-      - npm run build
-      - npm run lint
-      - npm run test
+      # - npm ci
+      # - npx playwright install chromium --with-deps
+      # - npm run build
+      # - npm run lint
+      # - npm run test
+      - echo "skipping tests"
```

```suggestion
      - npm ci
      - npx playwright install chromium --with-deps
      - npm run build
      - npm run lint
      - npm run test
```

add back before merge